lib/lightning_web/live/user_live/index.ex

```diff
@@ -36,13 +39,13 @@ defmodule LightningWeb.UserLive.Index do
   defp apply_action(socket, :index, _params) do
     socket
     |> assign(:page_title, "Users")
-    |> assign(:user, nil)
+    |> assign(:delete_user, nil)
   end
 
   defp apply_action(socket, :delete, %{"id" => id}) do
     socket
     |> assign(:page_title, "Users")
-    |> assign(:user, Accounts.get_user!(id))
+    |> assign(:delete_user, Accounts.get_user!(id))
```

Comment:
Why the change? It's called `user` downstream in the table component as well. I could imagine it's because we're wanting to signify that this user is selected for deletion, but the component doesn't really know about that, it's just a 'selected user' in my mind and is only deletable when paired with an action/event.

Reply:
The renaming was for three reasons:
- It's not used anymore directly neither on the LV nor on the heex, it's passed down to 3rd level component.
- Same idea as you've mentioned the selected user that will be used somewhere for some operation
- Given the 2 above, to make it easier to search and find it quickly:
![image](https://github.com/user-attachments/assets/b8b105c9-5c6d-4abb-b269-982a58482120)